### PR TITLE
feat: added label for nulling instancesV2 interface

### PR DESCRIPTION
**Pull request by jtherin**

### CODE REVIEW

File: scaleway/instances.go [hunk 1]
```diff
@@ -279,6 +283,10 @@ func (i *instances) InstanceShutdown(ctx context.Context, node *v1.Node) (bool,
 	var instance *scwinstance.Server
 	var err error
 
+	if _, ok := node.Annotations[nodeLabelDisableLifeCycle]; ok {
+		return true, nil
```
Comment (Sh4d1) [inline scaleway/instances.go:287]:
```suggestion
		return false, nil
```

File: scaleway/instances.go [hunk 2]
```diff
@@ -304,6 +312,10 @@ func (i *instances) InstanceMetadata(ctx context.Context, node *v1.Node) (*cloud
 	var instance *scwinstance.Server
 	var err error
 
+	if _, ok := node.Annotations[nodeLabelDisableLifeCycle]; ok {
+		return &cloudprovider.InstanceMetadata{}, nil
```
Comment (Sh4d1) [inline scaleway/instances.go:316]: We should return the node addresses here

File: scaleway/loadbalancers.go [hunk 3]
```diff
@@ -375,6 +378,11 @@ func (l *loadbalancers) GetLoadBalancerName(ctx context.Context, clusterName str
 func extractNodesExternalIps(nodes []*v1.Node) []string {
 	var nodesList []string
 	for _, node := range nodes {
+		if address, ok := node.Annotations[nodeAnnotationNodePublicIP]; ok {
```
Comment (Sh4d1) [inline scaleway/loadbalancers.go:381]: Shouldn't we fallback to internal IP if external IP does not exist?

**Review by Sh4d1 — CHANGES_REQUESTED** [review]

File: scaleway/loadbalancers.go (hunk 3 above)
Comment (jtherin) [reply scaleway/loadbalancers.go:381]: Not a big fan ATM, will depend of VPC, ipv6, etc...

**Review by Sh4d1 — APPROVED** [review]
LGTM

**Pull request merged**